packages/bus-process-iterator/lib/ActorProcessIterator.ts

```diff
@@ -0,0 +1,89 @@
+import type { LogicalJoinType } from '@comunica/bus-rdf-join';
+import type { IAction, IActorArgs, IActorOutput, IActorTest, Mediate } from '@comunica/core';
+import { Actor } from '@comunica/core';
+import type { IActionContext } from '@comunica/types';
+import type * as RDF from '@rdfjs/types';
+import type { AsyncIterator } from 'asynciterator';
+import type { types } from 'sparqlalgebrajs/lib/algebra';
+
+/**
+ * A comunica actor for process-iterator events.
```

Comment:
Just thinking about naming.

When I read this, my first thought also goes to a process at OS-level. Like a Node.js process, or a Firefox process.
But maybe this is just me. But if you agree that naming could be improved, something like `transform-iterator` could be an alternative.